test/torchaudio_unittest/functional/functional_impl.py

```diff
@@ -13,6 +13,35 @@
 
 
 class Functional(TestBaseMixin):
+    def _test_resample_waveform_accuracy(self, up_scale_factor=None, down_scale_factor=None,
+                                         resampling_method="sinc_interpolation", atol=1e-1, rtol=1e-4):
```

Comment:
Could you try and see if `atol=1e-1` can be tightened?

Reply:
It doesn't look like it